ups/verify_ap_hits2015.table

```diff
@@ -0,0 +1 @@
+setupRequired(obs_decam)
```

Comment:
This goes beyond my knowledge of EUPS, but don't you also need a `.cfg` file? Most packages have one.